[update] Limitations

Rajakavitha1 opened the pull request:
# Description

Fixed the broken link

# Links

Fixes https://github.com/timescale/docs/issues/1130

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [x] Is the content free from typos?
- [x] Does the content use plain English?
- [x] Does the content contain clear sections for concepts, tasks, and references?
- [x] Are procedure and highlight tags used appropriately?
- [x] Has the index been updated appropriately?
- [x] Have any images been uploaded to the correct location, and are resolvable?
- [x] Are all links provided in reference style, and resolvable?
- [x] If the page index was updated, are redirects required
      and have they been implemented?
- [x] Have you checked the built version of this content?

charislam commented:
Hi @Rajakavitha1, this is totally my fault, when I moved the distributed hypertables page, I seem to have made the replication section vanish. So the link didn't actually give you the information it claimed! Should be fixed now but you should give the added section a read to make sure I didn't make any mistakes

Loquacity commented:
Can we get the Vale errors fixed before merge, please?

Rajakavitha1 commented:
Hi @charislam  and @Loquacity I incorporated teh changes that @charislam  committed and rewrote to avoid the vale errors. Please take a look and state your approval before the merge.

charislam commented on timescaledb/how-to-guides/distributed-hypertables/about-distributed-hypertables.md:
```diff
@@ -60,6 +60,17 @@ This can affect queries that span the two different partitioning configurations.
 down][limitations-pushing-down].
 
+## Replicating distributed hypertables
+To replicate distributed hypertables at the chuck level configure the
```
```suggestion
To replicate distributed hypertables at the chunk level, configure the
```
